app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -287,7 +287,6 @@ def ordering_scope(scope, ordering_params)
         sort_by = ordering_params[:sort_by] || 'started_at'
         sort_by = 'foreman_tasks_tasks.' + sort_by if sort_by == 'started_at'
         sort_order = ordering_params[:sort_order] || 'DESC'
-        scope = scope.select("foreman_tasks_tasks.*, coalesce(ended_at, current_timestamp) - coalesce(coalesce(started_at, ended_at), current_timestamp) as duration")
```

Comment:
Don't we use the duration somewhere? :thinking:

Reply:
I thought so too, but couldn't find it anywhere. Guess it is really there just for the sorting

Reply:
and as @jlsherrill is pointing out, it will be there already added by the `resource_scope_for_index`